cyberbrain/value_stack.py

```diff
@@ -187,6 +268,30 @@ def tos1(self):
     def tos2(self):
         return self._tos(2)
 
+    @staticmethod
```

Comment:
This method probably should live outside of the class

Reply:
Also, `merge_stack_item` feels a more appropriate name

Reply:
I had this before but you mentioned in https://github.com/laike9m/Cyberbrain/pull/132#discussion_r757279347 to change it

Reply:
Yes, because back then we were talking about having a single StackItem class

Reply:
Changed